doc/sphinx/source/ops/broadcast_distributed.rst

```diff
@@ -0,0 +1,46 @@
+.. broadcastdistributed.rst:
+
+#####################
+BroadcastDistributed
+#####################
+
+.. code-block:: cpp
+
+   BroadcastDistributed // Collective operation
+
+
+Description
+===========
+
+Broadcast values from one process or device (root) to the rest processes or 
+devices of the communicator.
+
```

Comment:
Maybe something like "Broadcasts values from a primary process or device to other processes or devices within the communication network"  ?